src/btree/col_modify.c

```diff
@@ -260,7 +260,7 @@ __wt_col_modify(WT_CURSOR_BTREE *cbt, uint64_t recno, const WT_ITEM *value, WT_U
  * __col_insert_alloc --
  *     Column-store insert: allocate a WT_INSERT structure and fill it in.
  */
-static int
+int
```

Comment:
Keep this function as a static function. This is not used outside of this file.

Reply:
Fixed.